docs/parameters.yaml

```diff
@@ -64,13 +64,13 @@ components: ["origin", "registry", "director"]
 ---
 name: IssuerKey
 description: |+
-  [Deprecated] A filepath to the file containing a PEM-encoded ecdsa private key which later will be parsed
+  [Deprecated] Use IssuerKeysDirectory instead. The key file set by this parameter will automatically move to IssuerKeysDirectory.
+  
+  A filepath to the file containing a PEM-encoded ecdsa private key which later will be parsed
   into a JWK and serves as the private key to sign various JWTs issued by this server.
 
   A public JWK will be derived from this private key and used as the key for token verification.
 type: filename
-deprecated: true
```

Comment:
I think we still want `-deprecated: true` even if there's no direct replacement -- this being set is what generates a warning for the user. Otherwise they may never know to check the documentation for a deprecation warning.

Reply:
seems like they need to be removed together, otherwise the Go Generate Check fails.
`panic: Parameter entry 'IssuerKey' is deprecated but missing 'replacedby' key. If there is no replacement, use 'none'`

Reply:
Ah, gotcha. Then in this case, I think we want to do what the param error suggests -- use `ReplacedBy: "none"`